### PR TITLE
Polish PCRGlob notebook

### PR DESCRIPTION
**Pull request by Peter9192**
- added logo to repo
- updated pcrglob notebook
- renamed example parameter set for pcrglob ("pcrglob_example_case" --> "pcrglob_rhinemeuse_30min")
- reverted to use SIF image instead of `docker://<docker image>` as it allows installation by sysadmin (see #159)

https://ewatercycle--167.org.readthedocs.build/en/167/examples/pcrglobwb.html

### CODE REVIEW

File: ewatercycle/models/pcrglobwb.py [hunk 1]
```diff
@@ -51,6 +52,13 @@ def _set_docker_image(self):
         }
         self.docker_image = images[self.version]
 
+    def _set_singularity_image(self):
+        images = {
+            "setters": "ewatercycle-pcrg-grpc4bmi-setters.sif",
+        }
+        image = CFG['singularity_dir'] / images[self.version]
+        self.singularity_image = str(image)
```
Comment (SarahAlidoost) [inline ewatercycle/models/pcrglobwb.py]:
```suggestion
    def _set_singularity_image(self, singularity_dir: Path):
        images = {
            "setters": "ewatercycle-pcrg-grpc4bmi-setters.sif",
        }
        image = singularity_dir / images[self.version]
        self.singularity_image = str(image)
```

File: ewatercycle/models/pcrglobwb.py [hunk 2]
```diff
@@ -197,7 +205,7 @@ def _start_container(self):
             )
         elif CFG["container_engine"] == "singularity":
             self.bmi = BmiClientSingularity(
-                image=f"docker://{self.docker_image}",
+                image=self.singularity_image,
```
Comment (SarahAlidoost) [inline ewatercycle/models/pcrglobwb.py]:
```suggestion
                image=self.singularity_image(CFG['singularity_dir']),
```

File: docs/examples/pcrglobwb.ipynb [hunk 3]
```diff
@@ -2,57 +2,130 @@
  "cells": [
```
Comment (SarahAlidoost) [inline docs/examples/pcrglobwb.ipynb:2]: <p>unused <code>import ewatercycle</code> can be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (SarahAlidoost) [inline docs/examples/pcrglobwb.ipynb:2]: <p>I found this kind of assignments <code>custom_parameter_set = parameter_set =</code> confusing and unnecessary. Can we just use <code>parameter_set</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (SarahAlidoost) [inline docs/examples/pcrglobwb.ipynb:2]: <p>directory should be <code>"./parameter-sets/pcrglobwb_rhinemeuse_30min/forcing"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='11'/>

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@Peter9192 looks nice, thanks. I could run the notebook. just minor comments on the notebook. I also suggested refactoring `def _set_singularity_image(self)` because if `container_engine` is docker, we don't need to provide `singularity_dir` in the config file. 
Also, please edit these two sections in System setup: 

- the pcrglob singularity command in  [Singularity](https://ewatercycle.readthedocs.io/en/latest/system_setup.html#id2).

- the name of the pcrglob parameterset in [Download example parameter sets](https://ewatercycle.readthedocs.io/en/latest/system_setup.html#download-example-parameter-sets).

File: docs/examples/pcrglobwb.ipynb (hunk 3 above)
Comment (Peter9192) [reply docs/examples/pcrglobwb.ipynb:2]: oops, that was a mistake. I intended to only write `custom_parameter_set = ParameterSet(...)`. I'd like it to keep the 'custom' in the name, though.

**Comment by Peter9192** [comment]
Thanks for the nice review @SarahAlidoost :-)
I implemented your suggestions. I also made another change to the model class (make forcing optional) in the last commit. I realized we had not done this yet, and I need it for the irrigation notebook.

**Review by SarahAlidoost — APPROVED** [review]
@Peter9192 thank you, looks good to me :)

**Pull request merged**